src/ghga_datasteward_kit/file_deletion.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Provides functionality to request file service data deleteion."""
+
+import logging
+from pathlib import Path
+
+import httpx
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+from ghga_datasteward_kit.utils import DELETION_TOKEN, load_config_yaml
+
+log = logging.getLogger(__name__)
+
+
+class FileDeletionConfig(BaseSettings):
+    """Config for calling the PCS file deletion endpoint"""
+
+    file_deletion_baseurl: str = Field(
+        default=...,
+        description=(
+            "Base URL under which the delete /files endpoint is available."
```

Comment:
```suggestion
            "Base URL under which the file deletion endpoint is available."
```

since it's configurable